lib/events/index.js

```diff
@@ -31,18 +31,28 @@ if (DISABLED_EVENT_TYPES) {
   });
 }
 
+const DISABLED_AFTER_TIMESTAMP = config.get('basket.sqs.disabled_after_timestamp');
+
 
 module.exports.handleEvent = function handleEvent(message) {
   logger.info('handleEvent', message);
   const messageHandler = messageHandlers[message.event];
-  if (messageHandler) {
-    return messageHandler(message);
-  } else {
+  if (! messageHandler) {
```

Comment:
I re-ordered this into a sequence of check-and-return-early rather than nested if/elses.